src/iscn/likerIdsAddresses.test.ts

```diff
@@ -0,0 +1,117 @@
+import axios from 'axios';
+import { getLikeWalletByLikerId, getLikerIdByWallet } from './likerIdsAddresses';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe('likerIdsAddresses', () => {
+  test('getLikeWalletByLikerId testnet', async () => {
+    mockedAxios.get.mockImplementation(() => Promise.resolve({
+      data: {
+        user: 'kuan85998',
+        displayName: 'kuan85998',
+        avatar: 'https://static.like.co/likecoin_de-portrait.jpg',
+        cosmosWallet: 'cosmos1twr950vdy4gr9aggq5lehden8m023e0aj26kpe',
+        likeWallet: 'like1twr950vdy4gr9aggq5lehden8m023e0apkx5zz',
+      },
+    }));
+    const res = await getLikeWalletByLikerId('kuan85998', { LIKE_CO_API_ROOT: 'https://api.rinkeby.like.co' });
```

Comment:
Using axios mock prevent us from testing `LIKE_CO_API_ROOT`, so maybe just skip tests related to `LIKE_CO_API_ROOT`